x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/store/mock_endpoint_result_list.ts

```diff
@@ -118,8 +124,8 @@ const endpointListApiPathHandlerMocks = ({
       };
     },
 
-    // Do policies referenced in endpoint list exist
-    // just returns 1 single agent policy that includes all of the packagePolicy IDs provided
+    // Do policies reference in endpoint list exist
+    // just returns 1 single agent policy that includes all the packagePolicy IDs provided
```

Comment:
I'm starting to think that you are somebody who actually reads the comments... 🦄  and even fixes them!  🦅 👁️

Reply:
😆 

Soooo... I am one of those that writes code comments. And - I'm so glad that @ashokaditya looks at them because he corrects all of my misspellings 😮